text/0000-Project-Goals-2024h2.md

```diff
@@ -0,0 +1,522 @@
+
+### docs-rs team
+| Goal                                                                  | Owner      | Notes |
+| ---                                                                   | ---        | --- |
+| *Collecting popular queries for review*                               |            |  |
+| ↳ [Feedback and testing](https://rust-lang.github.io/rust-project-goals/2024h2/rustdoc-search.html#ownership-and-team-asks)   | [Michael Howell][] |  |
+
+### lang team
+| Goal                                                                                                                                  | Owner            | Notes              |
+| ---                                                                                                                                   | ---              | ---                |
+| *Design meeting*                                                                                                                      |                  |                    |
+| ↳ [Patterns of empty types](https://rust-lang.github.io/rust-project-goals/2024h2/Patterns-of-empty-types.html#ownership-and-team-asks)                                                       | [@Nadrieril][]       |                    |
+| ↳ ["Stabilizable" prototype for expanded const generics](https://rust-lang.github.io/rust-project-goals/2024h2/min_generic_const_arguments.html#ownership-and-team-asks)                      | [Boxy][]         | Up to 1, if needed |
+| ↳ [Experiment with relaxing the Orphan Rule](https://rust-lang.github.io/rust-project-goals/2024h2/Relaxing-the-Orphan-Rule.html#ownership-and-team-asks)                                     | ![Help wanted][] | Up to 1, if needed |
+| ↳ [Ergonomic ref-counting](https://rust-lang.github.io/rust-project-goals/2024h2/ergonomic-rc.html#ownership-and-team-asks)                                                                   | [Jonathan Kelley][]      |                    |
+| *Discussion and moral support*                                                                                                        |                  |                    |
+| ↳ [Const traits](https://rust-lang.github.io/rust-project-goals/2024h2/const-traits.html#ownership-and-team-asks)                                                                             | [Deadbeef][]       |                    |
+| *Lang-team experiment*                                                                                                                |                  |                    |
+| ↳ [Expose experimental LLVM features for automatic differentiation and GPU offloading](https://rust-lang.github.io/rust-project-goals/2024h2/Rust-for-SciComp.html#ownership-and-team-asks)   | [Manuel Drehwald][]          | (approved)         |
+| *Org decision*                                                                                                                        |                  |                    |
```

Comment:
@nikomatsakis
I have some concerns on this one and would like to address those concerns before agreeing to this project goal.

Reply:
The "org decision" specifically?

Reply:
No, off-by-one error on the line the feedback was meant for. "Expose experimental LLVM features for automatic differentiation and GPU offloading".

Reply:
This is already an approved lang experiment. (It was discussed in the triage meeting on May 1.) @joshtriplett are you objecting to the experiment or specifically to its inclusion as a project goal?

Reply:
> This is already an approved lang experiment. (It was discussed in the triage meeting on May 1.) @joshtriplett are you objecting to the experiment or specifically to its inclusion as a project goal?

The latter. As an experiment we make almost zero commitments beyond that we'll take a look at the results of the experiment. As a project goal we're establishing a stronger team consensus on the requirements and axioms and similar stated in the project goal, so we need to actually have a team consensus on the contents of the goal we accept.